src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateConstants.cs

```diff
@@ -15,6 +16,7 @@ public sealed class PipelineTemplateConstants
         public const String Clean = "clean";
         public const String Container = "container";
         public const String ContinueOnError = "continue-on-error";
+        public const String CompositeOutputs = "composite-outputs";
```

Comment:
revert this file (see comment futher below)

Reply:
+1, otherwise you will need to port this back to server code